README.md

```diff
@@ -37,6 +37,39 @@ Somewhere in your code, you should setup the client:
   (Thread/sleep 100))
 ```
 
+### Ring Middleware
+
+This library also has comes with a ring middleware to capture HTTP requests.
+See `com.unbounce.dogstatsd.ring` for more information.
+
+The middleware provides these metrics:
+
+- http.1xx  counter of 1xx responses
+- http.2xx  counter of 2xx responses
+- http.3xx  counter of 3xx responses
+- http.4xx  counter of 4xx responses
+- http.5xx  counter of 5xx responses
+- http.count     counter for total requests
+- http.exception counter for exceptions raised
+- http.duration  histogram of request duration
```

Comment:
I wanted to see if there were statsd http clients that provided a standard set of metrics to mimic, but I couldn't -- so I chose these 🤷‍♂️